bin/find_wavelengths_for_satpy.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020 Simon Proud
+
+# Author(s):
+
+#   simon.proud@physics.ox.ac.uk <simon.proud@physics.ox.ac.uk>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Please fill the empty lines with `#`. And I believe the current preferred copyright is `Pytroll developers`.

Reply:
Agree, can you fix @simonrp84 ?